source_code/aux_mcu_v4/src/main.c

```diff
@@ -222,3 +228,77 @@ int main(void)
 
 	/* Insert application code here, after the board has been initialized. */
 }
+
+#undef DEBUG_LOG_DISABLED
+#if !defined DEBUG_LOG_DISABLED
```

Comment:
Mistake. Will remove #undef. It was there because in the code I had a the time there was an explicit define in a different file instead of just in the makefile.